pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -912,11 +912,15 @@ func (r *ReconcileConstraintTemplate) manageVAP(ctx context.Context, ct *v1beta1
 		}
 		logger.Info("get VAP", "vapName", vapName, "currentVap", currentVap)
 
-		transformedVap, err := r.transformTemplateToVAP(unversionedCT, vapName, logger)
-		if err != nil {
-			logger.Error(err, "transform to VAP error", "vapName", vapName)
-			err := r.reportErrorOnCTStatus(ctx, ErrCreateCode, "Could not transform to VAP object", status, err)
-			return err
+		transformedVap, err1 := r.transformTemplateToVAP(unversionedCT, vapName, logger)
```

Comment:
nit: more descriptive err name


```suggestion
		transformedVap, transformErr := r.transformTemplateToVAP(unversionedCT, vapName, logger)
```